markup-filter-theme.md

```diff
@@ -36,7 +36,7 @@ The following aliases are supported:
 
 Alias | Description
 ------------- | -------------
-`@jquery` | Reference to the jQuery library (v3.4.0) used in the back-end. (JavaScript)
+`@jquery` | Reference to the jQuery library (latest available version) used in the back-end. (JavaScript)
```

Comment:
keep the version number please, it's a good quick reference for developers to know what version we use.